com.ktechpit.ultimate-media-downloader.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: com.ktechpit.ultimate-media-downloader
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
```

Comment:
doesn't it ship with Qt itself? Since it's proprietary.

Reply:
No it doesn't, it will use shared libraries from kde platform runtime. Is there any issue with that? 
Please let me know.